### DISCUSSION TITLE
Try to download Prisma / Next js fullstack tutorial

### DISCUSSION THREAD

**Discussion by deltaerp** [prisma/prisma#2379] [Q&A] [answered]
Hi,
I am trying to download the prisma / next js full stack tutorial but it says "Permission denied". Could you please be kind to give me access to this tutorial.

Best regards,
JA

**Comment by ryands17** [comment, marked as answer]
Hey @deltaerp :wave: 
Are you taking about [this](https://github.com/prisma/prisma-examples/tree/master/typescript/rest-nextjs) repository?

If not, could you share the link that you are referring to?

**Comment by deltaerp** [comment]
Hi Ryan,
Yes.

Regards,
JA

Sent from Mail for Windows 10

**Comment by ludralph** [comment]
Hi @deltaerp,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
